sql/ddl_api.sql

```diff
@@ -192,15 +192,27 @@ CREATE OR REPLACE FUNCTION detach_data_node(
 ) RETURNS INTEGER
 AS '@MODULE_PATHNAME@', 'ts_data_node_detach' LANGUAGE C VOLATILE;
 
--- Block new chunk creation on a data node for a distributed hypertable. NULL hypertable means it will block
--- chunks for all distributed hypertables
-CREATE OR REPLACE FUNCTION block_new_chunks(data_node_name NAME, hypertable REGCLASS = NULL, force BOOLEAN = FALSE) RETURNS INTEGER
-AS '@MODULE_PATHNAME@', 'ts_data_node_block_new_chunks' LANGUAGE C VOLATILE;
-
--- Reallow chunk creations on a blocked data node for a distributed hypertable. NULL hypertable means it will
--- allow chunks for all distributed hypertables
-CREATE OR REPLACE FUNCTION allow_new_chunks(data_node_name NAME, hypertable REGCLASS = NULL) RETURNS INTEGER
-AS '@MODULE_PATHNAME@', 'ts_data_node_allow_new_chunks' LANGUAGE C VOLATILE;
+-- Cordon a data node across all distributed hypertables or a specific
+-- one if a hypertable is specified. A cordoned node will not be used
+-- for queries or have any new chunks placed on it. The force
```

Comment:
I think this is slightly misleading, I guess cordoned chunk still can be used for queries, right?  Only new chunks will not be created on it but old data should still be accessible

Reply:
It shouldn't be used for queries, but that will be implemented separately. I can remove this comment in the meantime if you think it is important.

Reply:
I am ok to keep it, but we should do it before the release I guess. Our docs don't mention that either